modules/chain-grabber/src/main/scala/org/ergoplatform/explorer/indexer/cache/ApiQueryCache.scala

```diff
@@ -0,0 +1,17 @@
+package org.ergoplatform.explorer.indexer.cache
+
+import dev.profunktor.redis4cats.RedisCommands
+
+trait ApiQueryCache[F[_]] {
+  def flushAll: F[Unit]
+}
```

Comment:
Why pieces of Cache API are distributed across different modules? Why not gather all of them in a separate module and then just depend on it where needed?